sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryBuilderTests/FullQueryTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.DigitalTwins.Core.QueryBuilder;
+using Azure.DigitalTwins.Core.Queries.QueryBuilders;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.DigitalTwins.Core.Tests.QueryBuilderTests
+{
+    public class FullQueryTests
+    {
+        [Test]
+        public void SelectSinglePropertyTests()
+        {
+            new AdtQueryBuilder()
```

Comment:
Unit tests should generally be testing just 1 condition and should only intentionally fail for 1 bug. I'd suggest separating these into 2 tests.